client/src/components/Questions/.AnswerInnerList copy.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import AnswerElement from './AnswerElement';
+import { getProductQuestions,
+  getAnswers,
+  addQuestion,
+  addAnswer,
+  markQuestionHelpful,
+  markAnswerHelpful,
+  reportQuestion,
+  reportAnswer,
+} from '../../utils';
+
+const AnswerContainer = styled.div`
```

Comment:
The styling in your section looks great in the browser!
-I suggest that the placeholder search bar test should be lower case
-I suggest making the search bar longer so that the placeholder text is fully visible